supportfiles/test.sh

```diff
@@ -20,32 +18,26 @@ EXITCODE=0
 
 # Always run nbtest in all TOPLEVEL_NB_FOLDERS, set EXITCODE to failure
 # if any run fails
-for folder in ${TOPLEVEL_NB_FOLDERS}; do
-    echo "========================================"
-    echo "FOLDER: ${folder}"
-    echo "========================================"
-    cd ${NOTEBOOKS_DIR}/${folder}
-    for nb in $(find . -name "*.ipynb"); do
-        nbBasename=$(basename ${nb})
-        # Skip all NBs that use dask (in the code or even in their name)
-        if ((echo ${nb}|grep -qi dask) || \
-            (grep -q dask ${nb})); then
-            echo "--------------------------------------------------------------------------------"
-            echo "SKIPPING: ${nb} (suspected Dask usage, not currently automatable)"
-            echo "--------------------------------------------------------------------------------"
-        elif (echo " ${SKIPNBS} " | grep -q " ${nbBasename} "); then
-            echo "--------------------------------------------------------------------------------"
-            echo "SKIPPING: ${nb} (listed in skip list)"
-            echo "--------------------------------------------------------------------------------"
-        else
-            cd $(dirname ${nb})
-            nvidia-smi
-            ${NBTEST} ${nbBasename}
-            EXITCODE=$((EXITCODE | $?))
-            rm -rf ${LIBCUDF_KERNEL_CACHE_PATH}/*
-            cd ${NOTEBOOKS_DIR}/${folder}
-        fi
-    done
+for nb in $(find repos/*/notebooks/* -name *.ipynb); do
```

Comment:
minor: the old code used a var to make it a little more self-documenting, but if you don't want to use a var you might consider adding a comment to explain to future maintainers what this `find` call is intended to find.

Reply:
I was actually thinking of the old code that did this: `for folder in ${TOPLEVEL_NB_FOLDERS};`, but I see your new code just uses one `find` to find all NBs under the `repos` dir.  A comment might be nice but not critical - totally up to you.